Tests/JWKx5cTests.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  JWKx5cTests.swift
+//  Tests
+//
+//  Created by Xaver Lohmüller on 12.10.18.
+//
+
+import XCTest
+@testable import JOSESwift
+
+class JWKx5cTests: XCTestCase {
+    func testBuildingJWKSetShouldNotFailIfCertificatesArePresent() {
```

Comment:
This test would probably fit nicely into [`JWKRSADecodingTests.swift`](https://github.com/xaverlohmueller/JOSESwift/blob/96fe42f8a12ed790b2637c870d5671b602a07af4/Tests/JWKRSADecodingTests.swift#L27).